Apps/Common/Test/TestWeakOperators.C

```diff
@@ -51,13 +51,10 @@ TEST(TestWeakOperators, Advection)
 
   Matrix EM_vec(2*2, 2*2);
   U[0] = 3.0; U[1] = 4.0;
-  // First component only
-  WeakOperators::Advection(EM_vec, fe, U, 1.0, 1, 2);
-  // Second component only
-  WeakOperators::Advection(EM_vec, fe, U, 2.0, 1, 2, 1);
-  const DoubleVec EM_vec_ref = {{15.0,  0.0, 22.0,  0.0},
+  WeakOperators::Advection(EM_vec, fe, U, 2.0);
+  const DoubleVec EM_vec_ref = {{30.0,  0.0, 44.0,  0.0},
```

Comment:
I'm just wondering why the test output changed. I'm sure all is fine since reg tests using Advection still pass...